monai/networks/trt_compiler.py

```diff
@@ -443,12 +446,11 @@ def _build_and_save(self, model, input_example):
             )
         else:
             if self.method == "onnx_dynamo":
-                dynamo = True
                 import torch_onnx
 
                 torch_onnx.patch_torch()
-            else:
-                dynamo = False
+                export_args["dynamo"] = True
```

Comment:
I think the `dynamo` parameter is not used in [this function](https://github.com/borisfom/MONAI/blob/9481d9f6a368c5cac836cca39e493bcc20a6532a/monai/networks/utils.py#L612). What's the purpose of this line?

Thanks,  
Bin

Reply:
Why, it's being passed over there  in **kwargs and goes to onnx.export(). By not passing it explicitly as non-kw-arg, I make sure the code will work with older Torch versions that do not have 'dynamo' arg in onnx.export() (unless method='onnx_export' is actually used)

Reply:
The torch.onnx.export function is using [torch_versioned_kwargs](https://github.com/borisfom/MONAI/blob/3d221cbf1d62167369640d910e00b11759796cd3/monai/networks/utils.py#L684) instead of `kwargs`. And if the logic goes through [this branch](https://github.com/borisfom/MONAI/blob/3d221cbf1d62167369640d910e00b11759796cd3/monai/networks/utils.py#L661). The `dynamo` parameter will not be passed through the `torch_versioned_kwargs`. Is this expected?

Thanks,  
Bin

Reply:
No, it's always use_trace branch.

Reply:
Got it, thanks.